Nbranch

Khanetmoi opened the pull request:
The first version of the "Microverse hello project".It displays "Hello Microverse!" written in red on a white page

Khanetmoi committed:
create index.html, style.css, gitingnore and test.md

Khanetmoi committed:
basic html structure and header  added to the html file

Khanetmoi committed:
color and font size added to the class header in the css file

Khanetmoi committed:
Update README.md

Khanetmoi committed:
Update README.md

Khanetmoi committed:
Create .gitignore

Khanetmoi committed:
Delete test.md

Khanetmoi committed:
Delete test.md

AnselemOdims reviewed (commented):
# Invalid Request 🛑 

Please make the right Pull Request link is submitted to the right project on your dashboard.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. Use the tag @AnselemOdims for a quicker reply.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

thecodechaser reviewed (commented):
## Hi @Khanetmoi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Since you have merged this Pull request into the main branch, You can open a new PR with all the required changes I have mentioned and also in the new PR description, Reference this PR Link, So that Other reviewers can see, What required changes you have to implement.

### NOTE:- Never merge the Pull request before Approval.

## Required Changes ♻️
- [x] Linter should be set up correctly for this repository.  Please set up linter for this project. You can use this template to add linter configuration to your project.[[Linters](https://github.com/microverseinc/linters-config/tree/master/html-css)]
- [x] Linter result should be green, this means that the last commit has a green checkmark (✅) or a summary of the passed checks is in the PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Khanetmoi commented:
Hi, thank you for your review, I've corrected my mistakes, added an HTML-CSS linter, and made sure that the linter results were green, the only problem is that I can't find a way to ask for a new review. Can you help me, please?
